Document pre-baked image (#19)
debezium/microservices-lab: vm-setup/README.md

# Debezium Microservices Lab Vagrant image

This is a source repository for [Vagrant image](https://app.vagrantup.com/debezium/boxes/microservices-lab) serving as a Debezium microservices lab environment.

## Image usage
There are two ways how to use the image

> * (recommended) Use provided `Vagrant` file. In this case just downlod the `Vagrantfile` into an arbitrary directory and issue `vagrant up` command.

Reviewer: ```suggestion
* (recommended) Use provided `Vagrant` file. In this case just download the `Vagrantfile` into an arbitrary directory and issue `vagrant up` command.
```

After a while the new virtual machine will be spawned and you could log to it via `vagrant ssh`.

> * (advanced) Create the envrionment form scratch using `vagrant init debezium/microservices-lab` command.

Reviewer: ```suggestion
* (advanced) Create the environment from scratch using `vagrant init debezium/microservices-lab` command.
```

In this case the user is responsible for configuration of networking, port-forwarding and capacity allocation.

## Building the image

> To build the image from scratch it is necessary to folow this procedure

Reviewer: ```suggestion
To build the image from scratch it is necessary to follow this procedure
```

> * Use `Vagrantfile.build`as the Vagrant configuration file.

Reviewer: How do I do this exactly?

Author: Unfortunately only copy /rename - I've not found any other way

* Start the virtual machine using `vagrant up`.
* Log into virtual machine using `vagrant ssh`.
* As `root` execute
```
$ dnf clean all

> # It is expected that the following command will fill the disk and generate an error

Reviewer: What's the purpose of this?

Author: It is for better compressibility of the disk image - recommendation from vagrant
